utils/mstpctl-utils-functions.sh

```diff
@@ -2,7 +2,7 @@ mstpctl_parse_ports()
 {
   while [ x"${1+set}" = xset ]
   do
-    # For compatibility: the `all' option.
+    # For compatibility: the 'all' option.
```

Comment:
I mean: If Debian makes changes or bug fixes to that file, we will no longer be able to simply `cp /lib/bridge-utils/bridge-utils.sh utils/mstpctl-utils-functions.sh` to update our version with their changes.
Not a big deal, just something to be aware of.